docs/upgrade-notes/update-cumulus_id-type-indexes-CUMULUS-3449.md

```diff
@@ -0,0 +1,176 @@
+
+- [PostgreSQL Documentation](https://www.postgresql.org/docs/13/app-psql.html)
+- [Tmux tutorial](https://www.linuxtrainingacademy.com/tmux-tutorial/)
+
+:::note
+The version 9.x psql will be installed. This will work for all our standard SQL but some meta-commands like \d may not work.
+:::
+
+Basic commands for running SQL commands
+
+```sh
+# Start a tmux session called CumulusUpgrade with a new window named Fix-DataTypes
+tmux new -s CumulusUpgrade -n Fix-DataTypes
+ 
+# Start a PosgreSQL client section on the writer instance of the database cluster and prompt for password
+psql -h <Endpoint for writer instance> -p <Port for database or 5432> -d <cumulus database name or postgres> -U <database admin user or postgres> -W
+  
+# Open a new window to use for monitoring
+<Ctrl>-b c
+ 
+# Switch between windows
+# Last window
+<Ctrl>-b l
+
+# Next Window
+<Ctrl>-b n
+ 
+# Detach from tmux session
+<Ctrl>-b d
+ 
+# List tmux sessions
+tmux ls
+
+# Reattach to the session, you will see the windows again
+tmux attach -t CumulusUpgrade
+
+# Exit the window and/or close/kill the session
+# Warning: Don't close/kill the session until the task is complete
+exit
+# or
+<Ctrl>-b x
+```
+
+## Upgrade Steps
+
+1. Quiesce ingest
+
+    Stop all ingest operations in Cumulus Core according to your operational procedures. You should validate
+    that it appears there are no active queries that appear to be inserting granules/files into the database
+    as a secondary method of evaluating the database system state:
+
+    ```text
+    select pid, query, state, wait_event_type, wait_event from pg_stat_activity where state = 'active';
+    ```
+
+    If query rows are returned with a `query` value that involves the tables, make sure ingest is halted
+    and no other granule-update activity is running on the system.
+
+    Note: In rare instances if there are hung queries that are unable to resolve, it may be necessary to
+    manually use psql [Server Signaling
+    Functions](https://www.postgresql.org/docs/13/functions-admin.html#FUNCTIONS-ADMIN-SIGNAL)
+    `pg_cancel_backend` and/or
+    `pg_terminate_backend` to end the queries.
+
+2. Login into EC2 instance with database access.
+
+    From AWS console, go to EC2, pick a `<prefix>-CumulusECSCluster` instance, click Connect, click Session Manager.
+    :::note Remember to take a note on which instance you run the commands. :::
+
+3. Install tmux and postgres client
+
+    ```sh
+    sudo yum install -y tmux postgresql.x86_64
+    ```
+
+    Once installed, a tmux session is started with two windows. The Cumulus database is connected to in each window
+    using the PostgreSQL client. The primary window is used for running the ALTER commands, while the secondary window
+    is used to monitor the database and alter statement. When the operator hits end of shift or is done monitoring for
+    the day, the tmux session can be detached from and reattached to at a later time.
+
+4. Run SQL Commands
```

Comment:
I think it would be helpful to add a line here that you should look at step 5 to monitor some of these commands. If I'm following the instructions line by line and am unfamiliar with tmux, which was the case for me, I might not see those instructions until I'm finished or almost finished with step 4.